Public/Sdk/Public/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -126,16 +143,31 @@ export function runQTest(args: QTestArguments): Result {
         const qTestContextInfoFile = Environment.getFileValue("[Sdk.BuildXL]qtestContextInfo");
         qTestContextInfoPath = qTestContextInfoFile.path;
     }
-     
+    // TODO: Renaming the internal flag passing from GBR, will remove the old one when the new one roll out from GBR
+    else if (Environment.hasVariable("[Sdk.BuildXL.CBInternal]qtestContextInfo")){
+        const qTestContextInfoFile = Environment.getFileValue("[Sdk.BuildXL.CBInternal]qtestContextInfo");
+        qTestContextInfoPath = qTestContextInfoFile.path;  
+    }
+    
+    let codeCoverageOption = getCodeCoverageOption();
     let changeAffectedInputListWrittenFile = undefined;
     let changeAffectedInputListWrittenFileArg = {};
-    if (qCodeCoverageEnumType === "DynamicCodeCov" && isChangeBasedCodeCoverage){
+    if (codeCoverageOption === CoverageOptions.DynamicChangeList){
         const parentDir = d`${logDir}`.parent;
         const leafDir = d`${logDir}`.nameWithoutExtension;
         const dir = d`${parentDir}/changeAffectedInput/${leafDir}`;
         changeAffectedInputListWrittenFile = p`${dir}/fileWithImpactedTargets.txt`;
         changeAffectedInputListWrittenFileArg = {changeAffectedInputListWrittenFile : changeAffectedInputListWrittenFile};
     }
+
+    let qCodeCoverageEnumType = undefined;

```

Comment:
>l [](start = 4, length = 1)

This can be shorten to:
```
let qCodeCoverageEnumType = Environment.getStringValue("[Sdk.BuildXL.CBInternal]CodeCoverageOption")
    || ((codeCoverageOption === CoverageOptions.DynamicChangeList || codeCoverageOption === CoverageOptions.DynamicFull)
         ? "DynamicCodeCov"
         : CoverageOptions.None.toString());
 
```